src/SearchPage.js

```diff
@@ -55,7 +55,7 @@ export default class SearchPage extends Component {
           this.setState({ isLoading: true})
           const response = await request.get(`https://pokedex-alchemy.herokuapp.com/api/pokedex?pokemon=${this.state.query}&sort=pokemon&direction=${this.state.sortOrder&this.state.currentPage}`);
```

Comment:
the sorting and pagination is not working because the url is set up wrong, take a look at the api page where it shows how to do the paging to figure out what is up.